modules/eslint-plugin/scripts/generate-overview.ts

```diff
@@ -5,7 +5,7 @@ import { rules } from '../src/rules';
 import { configs } from '../src/configs';
 
 const prettierConfig = resolveConfig.sync(__dirname);
-const OVERVIEW = './projects/ngrx.io/content/guide/eslint-plugin/overview.md';
+const OVERVIEW = './projects/ngrx.io/content/guide/eslint-plugin';
```

Comment:
We need to specify the full path here, otherwise `readFileSync` will fail:

```suggestion
const OVERVIEW = './projects/ngrx.io/content/guide/eslint-plugin/index.md';
```

Reply:
@markostanimirovic sorry for this, didn't notice this was required. Fixed ✔️